services/QuillLMS/app/models/concept_result.rb

```diff
@@ -29,6 +29,8 @@ class ConceptResult < ApplicationRecord
   validates :concept, presence: true
   validates :activity_session_id, presence: true
 
+  has_one :student_response_concept_result, dependent: :destroy
```

Comment:
This is to avoid adding a foreign key column to a 1.4B row table, right? 👍

Reply:
I hadn't fully thought that through, actually, but that is a very good reason not to put a ForeignKey constraint in there.